src/main/java/apoc/text/Strings.java

```diff
@@ -303,11 +303,12 @@ public String rpad(@Name("text") String text, @Name("count") long count, @Name(v
     }
 
     @UserFunction
-    @Description("apoc.text.format(text,[params]) - sprintf format the string with the params given")
-    public String format(@Name("text") String text, @Name("params") List<Object> params) {
+    @Description("apoc.text.format(text,[params],language) - sprintf format the string with the params given")
+    public String format(@Name("text") String text, @Name("params") List<Object> params, @Name(value = "language",defaultValue = "") String lang) {
```

Comment:
non si può mettere dentro `defaultValue`:
```suggestion
    public String format(@Name("text") String text, @Name("params") List<Object> params, @Name(value = "language",defaultValue = "en") String lang) {
```
e togliere l'if sotto?

Reply:
si